foundation/foundation-configuration-store/src/test/java/util/ConfigStoreInitTestHelper.java

```diff
@@ -0,0 +1,138 @@
+package util;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Configuration::Store
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.configuration.store.ConfigLoader;
+import com.boozallen.aissemble.configuration.store.Property;
+import io.quarkus.runtime.Startup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yaml.snakeyaml.DumperOptions;
+import org.yaml.snakeyaml.Yaml;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.spi.CDI;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * ConfigStoreInit loads configuration properties as application starts up
+ */
+@Startup
+@ApplicationScoped
+public class ConfigStoreInitTestHelper {
+    private static final Logger logger = LoggerFactory.getLogger(ConfigStoreInitTestHelper.class);
+
+    @PostConstruct
+    void init() {
+        prepareTestEnv();
+        logger.info("Initialize store configuration properties...");
+        String baseUri = System.getenv("BASE_URI");
+        String environmentUri = System.getenv("ENVIRONMENT_URI");
+        try {
+            ConfigLoader configLoader = CDI.current().select(ConfigLoader.class,new Any.Literal()).get();
+            Set<Property> properties = null;
+            if (environmentUri != null && !environmentUri.isEmpty()) {
+                properties = configLoader.loadConfigs(baseUri,environmentUri);
+            } else if (baseUri != null && !baseUri.isEmpty()){
+                properties = configLoader.loadConfigs(baseUri);
+            } else {
+                logger.warn("No properties are loaded. To load properties, you need to set the environment variable BASE_URI and ENVIRONMENT_URI");
+            }
+            if (properties != null) {
+                configLoader.write(properties);
+            }
+        } catch (Exception e) {
+            logger.error("Error loading properties:");
+            e.printStackTrace();
```

Comment:
A: More `printStackTrace` calls in this file that should be cleaned up (this one and one below)